cvlization/torch/training_pipeline/doc_ai/huggingface/donut/model.py

```diff
@@ -167,13 +193,11 @@ def __init__(
         self.add_tokens([self.task_start_token, self.prompt_end_token] + (self.additional_tokens or []))
         self.prompt_end_token_id = processor.tokenizer.convert_tokens_to_ids(self.prompt_end_token)
 
-    @property
-    def additional_tokens(self):
```

Comment:
Aimed to parse these from the dataset rather than hardcode.

Reply:
good point, `additional_tokens` is more closely coupled with the task-specific dataset, rather than the model